utils/java/mvn.go

```diff
@@ -21,7 +22,7 @@ const (
 	mavenDepTreeJarFile    = "maven-dep-tree.jar"
 	mavenDepTreeOutputFile = "mavendeptree.out"
 	// Changing this version also requires a change in MAVEN_DEP_TREE_VERSION within buildscripts/download_jars.sh
-	mavenDepTreeVersion = "1.0.2"
+	mavenDepTreeVersion = "1.0.3"
```

Comment:
```suggestion
	mavenDepTreeVersion = "1.1.0"
```
latest version is 1.1.0, is it require to be 1.0.3? don't forget to run download_jars.sh if changing.

Reply:
I guess 1.1.0 will work as well, what is download_jars.sh ?

Reply:
run locally https://github.com/jfrog/jfrog-cli-core/blob/master/buildscripts/download-jars.sh